src/python/aim/_sdk/sequence.py

```diff
@@ -301,6 +301,13 @@ def _check(self, query, query_cache, *, aliases=()) -> bool:
         query_params.update({cp: c_proxy for cp in container_alias_names})
         return query.check(**query_params)
 
+    def delete(self):
+        del self._data_loader()[(self._ctx_idx, self._name)]
+        del self._container_tree[(KeyNames.SEQUENCES, self._ctx_idx, self._name)]
+
+        self._info.empty = True
+        self._info.next_step = 0
+
```

Comment:
Suggestion: set a flag which will prevent tracking on the deleted Sequence. Technically, the Python object still lives, and the track() function call is possible.

Reply:
I think there's no need to prevent tracking on a deleted sequence. In a deleted state it'll be empty, and tracking will be done from a step 0, the same as for newly initialized sequences